src/components/KInput/KInput.vue

```diff
@@ -35,6 +35,7 @@
       <!-- TODO: [beta] change input class to text-input -->
       <input
         v-bind="modifiedAttrs"
+        :aria-describedby="helpTextId"
```

Comment:
if we have the `for` attribute on the label, tying it to the input, this shouldn't be needed?

Reply:
It's a requirement outline in one of the accessibility findings from Level Access evaluation. Basically it mostly applies to error messages related to input errors.

Reply:
Then the attribute should be on the helpText container, not on the input

Reply:
No, the input is described by the help text element.

Reply:
I'm not talking about the 1:1 attribute name.

Reply:
I'm sorry, I'm confused.. but I see that you resolved this thread - happy to provide more context if needed